### PR TITLE
[feat] Add community calendar

### LINKED ISSUE #3036: Add Community Calendar page
**Describe the change you'd like to see**
Now that we have a new Google Calendar for Community Events, would be good to have a page like `http://knative.dev/community/calendar/`

### PR DESCRIPTION
**Pull request by csantanapr**
Closes #3036 

Adds the Community Calendar page.

### CODE REVIEW

File: community/calendar/_index.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: "Knative Community Calendar"
+linkTitle: "Community Calendar"
+weight: 29
+showlandingtoc: "false"
+type: "docs"
+---
+
+Attend events on our Community Calendar.
```
Comment (evankanderson) [inline community/calendar/_index.md:9]: Do you want to add details about what's in the calendar; i.e. opportunities to learn about Knative and meet other Knative users? You might also mention that events need not be organized by the Knative organization to be added to the calendar.

**Review by evankanderson — COMMENTED** [review]
Thanks for filling this in!

File: community/calendar/_index.md (hunk 1 above)
Comment (csantanapr) [reply community/calendar/_index.md:9]: Thank you I was looking for type of info to put here @evankanderson 

Should we add something if you want to make us aware of an event to add to the Calendar email us (should I use steering email?), posted in #communtiy Slack channel, or mention KnativeProject@ in Twitter ?

**Comment by csantanapr** [comment]
@evankanderson this is ready when you have a chance to review it again.

**Review by evankanderson — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**